### DISCUSSION TITLE
Is <route_name>.<http_method>.ts works?

### DISCUSSION THREAD

**Discussion by yjl9903** [nitrojs/nitro#202] [Q&A]
Hello!

I see the example [here](https://github.com/unjs/nitro/tree/main/examples/api-routes/api) using `test.post.ts` and `test.get.ts`.

Does this means that `test.post.ts` will automatically handle the POST request to `/test` and `test.get.ts` will handle the GET request. There is nothing about it at the [docs](https://nitro.unjs.io/guide/routing.html).

However, I tried it and `test.post.ts` still handle the GET request, and I must check the method name first.

**Comment by pi0** [comment]
Hi @yjl9903. yes it is supposed to do this! Can you please share your repository or create a minimal reproduction for the error? 🙏🏼

**Comment by yjl9903** [comment]
Reproduction repo is [here](https://github.com/yjl9903/starter-nitro).

Maybe the reason is that I use the `index.get.ts` and `index.post.ts`?

If I change them to `[...].get.ts` and `[...].post.ts`, it all works. And `[...].ts` has higher priority than `index.ts`.
